voice/methods/about.md

```diff
@@ -12,8 +12,13 @@ Some of the URLs in this documentation contain placeholders for values that your
 
 ## REST API Reference Index
 
-| Resource                                                              | Description                                          |
-|:----------------------------------------------------------------------|:-----------------------------------------------------|
-| [`/calls`](calls/postCalls.md)                                        | The Calls resource lets you place phone calls.       |
-| [`/calls/{callId}`](calls/postCallsCallId.md)                         | The CallsId resource lets you redirect active calls. |
-| [`/calls/{callId}/recording`](calls/putCallsCallIdRecording.md)       | Pause or resume recording on an active call.         |
+| Resource                                                                                              | Description                                                        |
+|:------------------------------------------------------------------------------------------------------|:-------------------------------------------------------------------|
+| [`/calls`](calls/postCalls.md)                                                                        | The Calls resource lets you place phone calls.                     |
+| [`/calls/{callId}`](calls/postCallsCallId.md)                                                         | The CallsId resource lets you redirect active calls.               |
```

Comment:
```
Redirect an active call to a new BXML
```

Reply:
How about
> The CallId resource lets you replace an active call's BXML
I still don't like the first half (i.e, `The CallsById resource`) :thinking:

Reply:
No need to name the resource. This is in a table, readers already know what resource this is talking about. My suggestion was to replace the whole description with what I said.